File: src/kyu_8/grasshopper_terminal_game_move_function/java/MoveTest.java
package kyu_8.grasshopper_terminal_game_move_function.java;

import org.junit.Test;
import static org.junit.Assert.assertEquals;

public class MoveTest {
    @Test
    public void basicTests() {
        assertEquals(8, Move.move(0, 4));
        assertEquals(15, Move.move(3, 6));
        assertEquals(12, Move.move(2, 5));
    }
}

File: src/kyu_8/grasshopper_terminal_game_move_function/js/moveTest.js
const Test = require('@codewars/test-compat');
const move = require("./move");

describe('move function', () => {
    it('basic tests', () => {
        Test.assertEquals(move(0, 4), 8);
        Test.assertEquals(move(3, 6), 15);
        Test.assertEquals(move(2, 5), 12);
    })
})